.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
```

Comment:
if you wanted to go all out on 'optimum CI' you could use https://github.com/actions/cache here (and then maybe report back how easy it was to set up)

Reply:
👍Made a note to have a look at that. Looks fun!